catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -45,6 +45,7 @@
 	private static SharedPreferences preferences;
 	public static final String EXCEPTION_FOR_REPORT = "EXCEPTION_FOR_REPORT";
 	private static boolean isCrashReportEnabled = BuildConfig.CRASHLYTICS_CRASH_REPORT_ENABLED;
+	public static boolean isInitialized = false;
```

Comment:
Afaik `@VisibleForTesting` only works for methods.
Please create getter and setter.